### ISSUE TITLE
After applying orientation "Family and Friends" project data is not visible: A11y_.NET 9 template_ Project _Orientation

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#27426]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG1.3.4; #A11ySev1; #iOS; #SH_.NET9template_iOS_Jan2025; #iphone; #.NET Core; #Device; #Orientation; #A11yWCAG2.1; #AINotInscope;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
iOS Version: 18.2

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Swipe till Hamburger menu and double tap on it, Projects options will screen will open.
4. Apply Orientation.
5. Observe that After applying orientation "Family and Friends" project data is not visible.

### Actual Result:
After applying orientation "Family and Friends" project data is not visible. Also newly created projects are not visible.

### Expected Result:
After applying orientation "Family and Friends" project data should be clearly visible. Also newly created projects should be clearly visible.

### User Impact:
Users with dexterity impairments will face difficulty that "Family and Friends" project or newly created wont be visible in landscape mode, data is missing to users.

### Attachment:

https://github.com/user-attachments/assets/bcd57740-28f8-453c-b486-4a86e422034e

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:03-06-25; #A11yFixed;

Verified the issue in the latest environment, issue is fixed hence closing the bug.

https://github.com/user-attachments/assets/530a8139-4c3c-428d-b055-1ef92d73f4a0
